docs/companion_guide.md

```diff
@@ -411,6 +411,31 @@ billion transactions per second (tps).
 | Schema / Data Representation | JSON Formatted |
 | Query Capabilities | [Not found in documentation] |
 
+| Peergos	   |  https://https://peergos.org/ |
+| -------------- | ---------------------------------------------------------- |
+| Description	   | Peergos is a decentralised protocol and open-source platform for storage, social media and applications |
+| Specification  | Proprietary specification |
+| Deployment     | Offered as a SaaS solution. Can be self hosted. |
+| Identity       | [PKI](https://book.peergos.org/security/pki.html) + [Centralized password management, with password based keypairs(password + username + public salt)](https://book.peergos.org/security/login.html) |
```

Comment:
identity keypairs are random. the password is use to retrieve your encrypted identity keypair from your DWN

Reply:
@ianopolous the Centralized password management, with password based keypairs(password + username + public salt) language was taken directly as language from https://book.peergos.org/security/login.html. If you think this wording should be updated, would welcome a PR describe what you think should be updated and why. We are happy to take a look and incorporate reasonable updates.

Reply:
Sounds like we need to make that clearer. :-) The "encrypted entry data" in that diagram, which contains the identity keypair and root capability to your filesystem) is stored only on your DWN. 
I can do a PR with these changes this week hopefully.